CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 [//]: # (comment: Don't forget to update lib/datadog/statsd.rb:DogStatsd::Statsd::VERSION when releasing a new version)
 
+## 4.8.0 / 2020.04.20
+
+* [Feature] Add support of more environment variables for tagging [#140][] by [@delner][]
+* [OTHER] Small optimizations [#139][] by [@tenderlove][]
+* [BUGFIX] Properly close UDPSocket before creating a new one [#143][] by [@zachmccormick][]
+* [OTHER] Refactor to make code more idiomatic [#138][] by [@kbogtob][]
+* [OTHER] Refactor to translate unit tests to rspec [#135][] by [@kbogtob][]
+* [OTHER] Bump rake requirement to >= 12.3.3 [#137][] by [@remeh][]
+
+## 4.7.1 / Do not use
+
+* 4.7.1 was released by mistake and is not to use
```

Comment:
It has not been released on the gem repo, only the tags has been created here. We could probably delete the tag and remove these links from the changelog.

Reply:
You're right!